artifactory/utils/golang/project/archive.go

```diff
@@ -53,3 +57,16 @@ func getFileName(sourcePath, filePath, moduleName, version string) string {
 
 	return filepath.Join(moduleID, filename)
 }
+
+func isExcludePathsByGitignore(path string, ignore gitignore.GitIgnore) bool {
```

Comment:
I suggest renaming this function to *isPathIgnoredByGit*.

Reply:
Done.